### PR TITLE
Code Snippets: 1.1-category-the-essence-of-composition

### PR DESCRIPTION
**Pull request by raulraja**
- [x] Gradle, Kotlin and Arrow build config
- [x] Type checking snippets with Ank
- [x] Includes code for 1.1-category-the-essence-of-composition

### CODE REVIEW

**Comment by hmemcpy** [comment]
It's alive!! 🎉 
I'll keep an eye on this repo (maybe I'll learn Kotlin/Arrow in the meantime :P)

File: src/main/ank/1.1-category-the-essence-of-composition.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+f . id == f
+id . f == f
+```
+```kotlin:ank:replace
+import arrow.core.compose
+import arrow.core.identity
+
+val f: (Int) -> Int = { it }
+val identity: (Int) -> Int = ::identity
+""
```
Comment (nomisRev) [inline src/main/ank/1.1-category-the-essence-of-composition.md:54]: Typo?

**Review by nomisRev — APPROVED** [review]

File: src/main/ank/1.1-category-the-essence-of-composition.md (hunk 1 above)
Comment (raulraja) [reply src/main/ank/1.1-category-the-essence-of-composition.md:54]: Nope, just an abuse of `ank:replace` that replaces the block with the last return statement, in this case `""`

**Comment by jasoet** [comment]
@raulraja could you close the issue related to this chapter?

**Pull request merged**